src/base/pages/Landing.vue

```diff
@@ -0,0 +1,352 @@
+<template>
+  <div class="landing bg-white q-pa-lg">
+    <p class="text-center">
+      <img
+        :src="logo"
+        class="logo"
+      >
+    </p>
+    <h1 v-t="'LANDING.TITLE'" />
+    <p>
+      <strong v-t="'LANDING.SUBTITLE1'" />
+    </p>
+    <p v-t="'LANDING.SUBTITLE2'" />
+
+    <p class="text-center q-py-lg">
+      <RouterLink
+        v-t="'SIGNUP.TITLE'"
+        class="button"
+        :to="{ name: 'signup' }"
+      />
+      <RouterLink
+        v-t="'JOINGROUP.BROWSE_GROUPS'"
+        class="button"
+        :to="{ name: 'groupsGallery' }"
+      />
+    </p>
+
+    <p class="text-center">
+      <QImg
+        :src="screenshots.pickups"
+        alt="karrot"
+        class="screenshot screenshot-fullwidth q-my-lg"
+      />
+    </p>
+
+    <p v-t="'LANDING.VISION1'" />
+
+    <div class="row inline-images">
```

Comment:
Maybe here one of the images could be of the Bike Kitchen. I have a collection of them here, free to use: https://cloud.disroot.org/s/DmKLL32s9ZxBCCo
I think `1 - bikeworkshop.jpg` could work fine

Reply:
Yes! I thought having some non-food saving group photos would be nice! I've added the bikeworkshop one now.